.github/dependabot.yml

```diff
@@ -5,6 +5,11 @@ updates:
     directory: '/'
     schedule:
       interval: daily
+  - package-ecosystem: github-actions
+    directory: '/'
+    target-branch: "release-0.12"
```

Comment:
This is something we should take care of when cutting release branches in future (since this section is present in `devel` now).

Reply:
Yeah at the same time we cut release branches we should update this config on devel to point to the new branch. It's only read from devel, so whatever ends on on the release branch itself doesn't matter.